### PR TITLE
Entity tag fix

### PR DESCRIPTION
**Pull request by aravindmd**
- Pass `id` as part of `orca` request that is required to fetch tags from `front50` during `DeleteEntityTagsTask`

### CODE REVIEW

File: swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/tagging/OrcaTaggingService.kt [hunk 1]
```diff
@@ -44,7 +44,8 @@ class OrcaTaggingService(
               OrcaJob(
                 type = tagRequest.type,
                 context = mutableMapOf(
-                  "tags" to listOf(tagRequest.id)
+                  "tags" to listOf(tagRequest.id),
+                  "id" to tagRequest.id
```
Comment (emjburns) [inline swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/tagging/OrcaTaggingService.kt]: So `id` needs to be the entity id, and `tags` is a list of strings of the tag id. These shouldn't be the same value I think?

Comment (aravindmd) [reply swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/tagging/OrcaTaggingService.kt]: fixed it to pass the correct tag and id , tested it locally and looks good now

File: swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/tagging/EntityTag.kt [hunk 2]
```diff
@@ -55,3 +56,9 @@ open class EntityTagRequest(
   open val application: String,
   open val description: String
 ) : TagRequest
+
+open class SwabbieEntityTag {
```
Comment (emjburns) [inline swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/tagging/EntityTag.kt]: you can probably just do 
`const val SWABBIE_ENTITY_TAG_NAME = "spinnaker_ui_alert:swabbie_deletion_candidate"`
in this file. What is the use of the class with a companion object?

Comment (aravindmd) [reply swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/tagging/EntityTag.kt]: i added it as a class with an companion object to access it in swabbie-orca/src/main/kotlin/com/netflix/spinnaker/swabbie/orca/tagging/OrcaTaggingService.kt

Comment (aravindmd) [reply swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/tagging/EntityTag.kt]: nvm , fixed it

**Pull request merged**